bin/support_stats_reporting.py

```diff
@@ -35,12 +35,21 @@ def create_dataframe_from_csv(filename: str):
     return dataframe
 
 
-def yesterdays_date():
+def previous_working_day():
     today = date.today()
-    yesterday = today - timedelta(days=1)
-    str_yesterday = str(yesterday)
 
-    return str_yesterday
+    diff = 1
+    if today.weekday() == 0:
+        diff = 3
+    elif today.weekday() == 6:
```

Comment:
What's this for, we aren't running this on a sunday are we?

Reply:
Agreed, we don't run this over the weekend, have made the change as suggested.